akka-docs/src/main/paradox/includes/actor-context.md

```diff
@@ -0,0 +1,15 @@
+
+<!--- #actor-context-typed-access --->
```

Comment:
I'm not sure pulling this out into an include is worth it, esp. since this include is specific to typed.

Reply:
I guess the idea was to pull in the same into actor.md?

Reply:
Yes, the idea from the ticket is the same content in both typed actor and persistence with this snippet.
* The file is to be shared for typed and untyped, hence the name convention.
* The snippet denotes actor/context/typed for a hierarchical organization of snippets
* The final snippet `-access` denotes the specific topic

I think I noted the addition in actor as a TODO still, needing to sort out where in the `actor.md` typed doc it should reside. Suggestions?

Reply:
@raboof the snippet is in two doc files now. Though let's consider if that is the best location in it.

Reply:
Duplicating this section in the docs might be confusing to the reader - if accessing the actor context is not different between regular typed actors and persistent typed actors, should we even document this separately for typed persistent actors at all? Could we just add a note to the persistence typed docs that this is the same, and link to the section in `actor.md`?

Reply:
WFM. The background on this was raised by a user familiar with untyped, migrating to typed.